app/Http/Controllers/AppointmentController.php

```diff
@@ -18,6 +18,17 @@ public function __construct(AppointmentService $service)
         $this->service = $service;
     }
 
+    public function index()
+    {
+        $appointments = $this->service->index();
+        return response()->json(IndexAppointmentResource::collection($appointments), Response::HTTP_OK);
+    }
+
+    public function approve()
```

Comment:
Send the id to the service from here. The service shouldn't access the request directly
```
public function approve($id){
        $appointment = $this->service->approve($id);
        ....
}
```

Reply:
done